lib/jekyll_pages_api_search/tags.rb

```diff
@@ -11,10 +12,13 @@ class SearchInterfaceTag < Liquid::Tag
 
     def render(context)
       site = context.registers[:site]
+      placeholder = Config.get(site, 'placeholder') ||
```

Comment:
Might be a good idea to do this with something more akin to a localization tool instead of keeping it in the config at some point, but this is not the PR for that.